src/containers/profile/Profile.jsx

```diff
@@ -0,0 +1,103 @@
+import React from "react";
+import { connect } from "react-redux";
+import { firestoreConnect } from "react-redux-firebase";
+import { compose } from "redux";
+import { Redirect } from "react-router-dom";
+import { Tabs, Tab, Container, Row, Col } from "react-bootstrap";
+import anonymousImage from "../../images/anonymousImage.png";
+import profileHeaderBG from "../../images/profileHeaderBG.png";
+import Header from "../../components/common/Header";
+import "./Profile.css";
+import AccountSettings from "./ProfileSections/AccountSettings/AccountSettings";
+import ProfilesList from "./ProfileSections/ProfilesList/ProfilesList";
+
+const Profile = (props) => {
+  const { profile, auth, id, isblocked } = props;
+
+  const extraComponents = () => {
+    return (
+      <div className="extraComponents">
+        <h1>
+          {profile.displayName ? profile.displayName : profile.fullName}
+          {profile.jobTitle && (
+            <span className="jobTitle">({profile.jobTitle})</span>
+          )}
+        </h1>
+        {(profile.city || profile.country) && (
+          <div style={{ fontSize: "1.25em" }}>
+            <b>From </b>
+            <span>
+              {profile.city && profile.country
+                ? `${profile.city}, ${profile.country}`
+                : profile.city && !profile.country
+                ? profile.city
+                : profile.country}
+            </span>
+          </div>
+        )}
+        {profile.brief && <p className="brief">{profile.brief}</p>}
+      </div>
+    );
+  };
+
+  // Protect the page from unauthorized access
+  if (!auth.uid) {
+    return <Redirect to="/signInUp" />;
+  }
+
+  if (profile) {
+    const profileImage = profile.imageURL ? profile.imageURL : anonymousImage;
+    return (
+      <div className="d-flex justify-content-center position-relative">
+        <Header
+          headerBG={profileHeaderBG}
+          imagePath={profileImage}
+          headerPage={"profileHeader"}
+          isProfile={true}
+          extraComponents={extraComponents()}
+        />
+        <div className="shadow-lg pt-3 profileContainer">
+          <Tabs
+            defaultActiveKey="accountSettings"
+            className="justify-content-center"
+          >
+            <Tab
+              eventKey="accountSettings"
+              title={<h4>Account Settings</h4>}
+              className="m-4"
+            >
+              <AccountSettings id={id} />
+            </Tab>
+            <Tab
+              eventKey="manageUsers"
+              title={<h4>Manage Users</h4>}
+              className="m-4"
+            >
+              <ProfilesList />
+            </Tab>
+          </Tabs>
+
+          {isblocked && <h5>this account has been blocked!!</h5>}
+        </div>
+      </div>
+    );
+  } else {
+    return <h4 className="container">loading profile...</h4>;
```

Comment:
I know it will show only when the page is loading but I don't think we should give the ```<h4 className="container">```  class name container and if you are using this for styling, it might affect other components' style; as we have a class called ```container``` in bootstrap